kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -43,6 +43,11 @@ data:
           enabled = false
           # When this is enabled (enabled = true), all kube events including normal events will be collected
 
+        [log_collection_settings.track_dropped_logs]
+          # Collect statistics about wether all container logs make it to Log Analytics (logs can be dropped if the data rate is too high)
+          enabled = true
+          # When this is disabled (enabled = false), statistics about log collection throttling will not be kept
```

Comment:
We might want to explain the stats are only for our internal telemetry

Reply:
Good point, I wasn't sure if I should add it to the configmap at all (if it's not there then we'll have a hard time referencing it later)

Reply:
(I'm open to suggestions either way)